spec/factories/legal_aid_applications.rb

```diff
@@ -27,80 +27,142 @@
     #        TRAITS TO SET STATE                          #
     #######################################################
 
+    trait :with_non_passported_state_machine do
+      before(:create) do |application|
+        application.state_machine = NonPassportedStateMachine.new
```

Comment:
These are going to be quite resource-intensive, can we replace it with another factory?

Creating 'real' objects in the database is slow and could slow down the test runs as it will be hit hundreds of times :/